mcmd/molgenis/security/permission_manager.py

```diff
@@ -0,0 +1,46 @@
+"""Client for the Permission Manager plugin of MOLGENIS (all versions)."""
+
+from urllib.parse import urljoin
+
+from mcmd.core.errors import McmdError
+from mcmd.molgenis import api
+from mcmd.molgenis.client import post_form, post
+from mcmd.molgenis.principals import PrincipalType, to_role_name
+from mcmd.molgenis.resources import ResourceType
+from mcmd.molgenis.security.permission import Permission
+
+
+def grant_permission(principal_type: PrincipalType,
+                     principal_name: str,
+                     resource_type: ResourceType,
+                     entity_type_id: str,
+                     permission: Permission):
+    """Grants permission on one (non-row) resource to one principal."""
+
+    data = {'radio-' + entity_type_id: permission.value}
```

Comment:
I'd expect all permission assignments to go through the permission API for servers that support it.